api/dps/api.proto

```diff
@@ -17,30 +17,43 @@ syntax = "proto3";
 option go_package = "github.com/optakt/flow-dps/api/dps";
 
 service API {
-  rpc GetFirst (GetFirstRequest) returns (GetFirstResponse) {}
-  rpc GetLast (GetLastRequest) returns (GetLastResponse) {}
-  rpc GetHeader (GetHeaderRequest) returns (GetHeaderResponse) {}
   rpc GetCommit (GetCommitRequest) returns (GetCommitResponse) {}
   rpc GetEvents (GetEventsRequest) returns (GetEventsResponse) {}
-  rpc GetRegisters (GetRegistersRequest) returns (GetRegistersResponse) {}
+  rpc GetFirst (GetFirstRequest) returns (GetFirstResponse) {}
+  rpc GetHeader (GetHeaderRequest) returns (GetHeaderResponse) {}
   rpc GetHeight (GetHeightRequest) returns (GetHeightResponse) {}
+  rpc GetLast (GetLastRequest) returns (GetLastResponse) {}
+  rpc GetRegisters (GetRegistersRequest) returns (GetRegistersResponse) {}
   rpc GetTransaction (GetTransactionRequest) returns (GetTransactionResponse) {}
+  rpc ListCollectionsForBlock (ListCollectionsForBlockRequest) returns (ListCollectionsForBlockResponse) {}
   rpc ListTransactionsForBlock (ListTransactionsForBlockRequest) returns (ListTransactionsForBlockResponse) {}
   rpc ListTransactionsForCollection (ListTransactionsForCollectionRequest) returns (ListTransactionsForCollectionResponse) {}
-  rpc ListCollectionsForBlock (ListCollectionsForBlockRequest) returns (ListCollectionsForBlockResponse) {}
 }
```

Comment:
I don't like this change. There was a certain logic to the order before. Ordering things alphabetically doesn't have value. The only reason I did that for command line parameters is because the flags package does it internally, so I wanted to be consistent with the command output.

Reply:
I feel like not having them alphabetically ordered might be confusing though, because of the sheer amount of endpoints/requests/responses that we have. Having them alphabetically ordered makes it easier to find what you're looking for IMO.

What was the order before? I feel like we'll just end up adding stuff at the end, as we add new indexes, rather than order them in a way that makes sense 🤔